airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -86,18 +82,6 @@ public AirbyteConnectionStatus check(final JsonNode config) throws Exception {
       }
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
-    } catch (final ConnectionErrorException ex) {
```

Comment:
For those unaware, the removal of returning an failed `AirbyteConnectionStatus` is so that it can continue to be bubbled up the stack and caught within the `IntegrationRunner` class where it will handle the failed `AirbyteConnectionStatus` message in a singular location, less code reuse as well